web/client/components/catalog/editor/MainForm.jsx

```diff
@@ -7,45 +7,74 @@
  */
 import React, { useState, useEffect } from 'react';
 import {get, find, isEmpty} from 'lodash';
+import { Button, InputGroup, FormControl as FC, Form, Col, FormGroup, ControlLabel, Alert } from "react-bootstrap";
+
 import Message from '../../I18N/Message';
 import HTML from '../../I18N/HTML';
-
 import {getConfigProp} from '../../../utils/ConfigUtils';
-
 import InfoPopover from '../../widgets/widget/InfoPopover';
-import { FormControl as FC, Form, Col, FormGroup, ControlLabel, Alert } from "react-bootstrap";
-
 import localizedProps from '../../misc/enhancers/localizedProps';
 import {defaultPlaceholder, isValidURL} from "./MainFormUtils";
 
+import IconRB from '../../../plugins/ResourcesCatalog/components/Icon';
```

Comment:
In this case it's better to use directly Glyphicon so we can remove Icon from this PR

Reply:
this is a custom branch where we backport that feature, i would prefer to avoid to have too many differences from the master branch. i would skip this